src/kbc_modules/offline_fs_kbc/mod.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2021 IBM Corp.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::kbc_modules::{KbcCheckInfo, KbcInterface};
+
+use anyhow::{anyhow, Result};
+use base64::decode;
+use openssl::symm::{decrypt, Cipher};
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+
+const KEYS_PATH: &str = "/etc/aa-offline_fs_kbc-keys.json";
```

Comment:
The filename doesn't have "kata" anywhere. If the attestation-agent is always part of kata-agent, it might be useful to add that. But I'm also OK with the current name.

Reply:
I think there's no need to associate with Kata.